creator-node/src/services/stateMachineManager/stateReconciliation/types.ts

```diff
@@ -4,9 +4,10 @@ export type SyncRequestAxiosData = {
   sync_type?: string
   immediate?: boolean
   forceResync?: boolean
+  forceWipe?: boolean
   timestamp?: string
   signature?: string
-  from_manual_route?: boolean
+  sync_even_if_disabled?: boolean
```

Comment:
sorry maybe a dumb question - but what does this do?

Reply:
this used to be a flag to know if the sync was triggered from the "manual route" (`/merge_primary_and_secondary`). if that was the case, it would perform the sync regardless of the enabled sync mode. I changed this so it's an optional flag that we can choose to have respect the `mergePrimaryAndSecondaryEnabled` env var or not